### PR TITLE
P-8311: add ms teams beta docs

### PR DESCRIPTION
**Pull request by andrewblaney**
TODO: add video for installing app (recorded, just need licence from screen studio to export it)

### CODE REVIEW

File: microsoft-teams/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+title: 'Microsoft Teams'
+---
+
+**Never miss another message from a customer in Microsoft Teams**
+
+<Warning>
+    Microsoft Teams integration is currently in beta. The integration won't currently be available for your workspace until we've enabled the feature flag.
```
Comment (preslavmihaylov) [inline microsoft-teams/installation.mdx:8]: I think `we've enabled the feature flag` is a bit too developer-focused. Perhaps we can do `until we've enabled it for your workspace`

File: microsoft-teams/installation.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    If you need any assistance or have feedback on the installation process please [reach out to us](mailto:help@plain.com)!
+
+</Warning>
+
+Our Microsoft Teams integration lets you sync messages from selected Teams channels to Plain and respond directly to customers from the platform.
+
+### Prerequisites
+
+For the integration you'll need:
+n
```
Comment (preslavmihaylov) [inline microsoft-teams/installation.mdx:19]: typo

**Review by presmihaylov — APPROVED** [review]

File: microsoft-teams/installation.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  and some functionality may change. There is a list below of known issues and limitations!
+</Warning>
+
+<Steps>
+    <Step title="Connect your Plain workspace to Teams">
+        - In Plain, navigate to **`Settings`** > **`Microsoft Teams`**.
+        - Press **`Connect to Teams`**.
+        - Follow the instructions in the pop-up window to sign in to your Microsoft account, make sure you've selected the correct
+        microsoft account if you have multiple.
+        - Once the authorisation is complete, you can download the Plain app for Teams (used in the next step).
```
Comment (ebach86) [inline microsoft-teams/installation.mdx:39]: Just so we keep the American spelling everywhere
```suggestion
        - Once the authorization is complete, you can download the Plain app for Teams (used in the next step).
```

File: microsoft-teams/installation.mdx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        <Warning>
+            The method of installation here will depend on how Teams is setup for your organisation. If you are having trouble
+            with setting this up, please get in touch so we can help!
+        </Warning>
+
+        #### Option 1 - Install via Teams App
+        - Navigate to the Apps tab
+        - Click Manage your apps
+        - Click Upload an app
+        - As a Teams admin, you should be able to 'Upload an app to your organisation's app catalogue' and upload the app you downloaded in the previous step
```
Comment (ebach86) [inline microsoft-teams/installation.mdx:53]:
```suggestion
        - As a Teams admin, you should be able to 'Upload an app to your organization's app catalogue' and upload the app you downloaded in the previous step
```

File: microsoft-teams/installation.mdx [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+If the user has a personal Teams integration (see below) then the message will be sent as that user:
+
+![User Reply](/public/images/ms-teams-int-user-reply.png)
+
+If the user does not have a personal Teams integration, we will send the message as the bot, impersonating the user, which will look something like the below:
+
+![Impersonated Reply](/public/images/ms-teams-int-impersonated-reply.png)
+
+## Authorise Teams users in Plain
```
Comment (ebach86) [inline microsoft-teams/installation.mdx:99]:
```suggestion
## Authorize Teams users in Plain
```

File: microsoft-teams/installation.mdx [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+Its recommended that anyone with a Teams seat follows the below steps to allow Plain to reply as their teams user. It also allows attributing any messages they send in the Teams channel to be associated with their Plain user.
```
Comment (ebach86) [inline microsoft-teams/installation.mdx:101]:
```suggestion
It's recommended that anyone with a Teams seat follows the below steps to allow Plain to reply as their teams user. It also allows attributing any messages they send in the Teams channel to be associated with their Plain user.
```

File: microsoft-teams/installation.mdx [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+Our Microsoft Teams integration lets you sync messages from selected Teams channels to Plain and respond directly to customers from the platform.
+
+### Prerequisites
+
+For the integration you'll need:
+n
+
+- A Teams account with at least one seat
+- The installation of the app for teams can only be perfomed by an admin as you may need to change settings to:
```
Comment (ebach86) [inline microsoft-teams/installation.mdx:22]:
```suggestion
- The installation of the app for Teams can only be performed by an admin as you may need to change settings to:
```

File: microsoft-teams/installation.mdx [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+  🚧 As we're still building the integration, some features may not work as expected or be missing,
+  and some functionality may change. There is a list below of known issues and limitations!
+</Warning>
+
+<Steps>
+    <Step title="Connect your Plain workspace to Teams">
+        - In Plain, navigate to **`Settings`** > **`Microsoft Teams`**.
+        - Press **`Connect to Teams`**.
+        - Follow the instructions in the pop-up window to sign in to your Microsoft account, make sure you've selected the correct
+        microsoft account if you have multiple.
```
Comment (ebach86) [inline microsoft-teams/installation.mdx:38]:
```suggestion
        Microsoft account if you have multiple.
```

**Review by presmihaylov — APPROVED** [review]

**Review by elisebach — APPROVED** [review]

File: microsoft-teams/installation.mdx [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+- A Teams account with at least one seat
+- The installation of the app for Teams can only be performed by an admin as you may need to change settings to:
+  - [Allow external apps to be installed](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/external-app-setup?view=o365-worldwide)
+  - [Allow external users to be added as guests to be invited to Teams](https://learn.microsoft.com/en-us/microsoft-365/solutions/collaborate-as-team)
+
+## Installation
+
+<Warning>
+  🚧 As we're still building the integration, some features may not work as expected or be missing,
+  and some functionality may change. There is a list below of known issues and limitations!
```
Comment (jesushernandez) [inline microsoft-teams/installation.mdx:29]: Can we link to the relevant section at the bottom?

File: microsoft-teams/installation.mdx [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+        - Follow the instructions in the pop-up window to sign in to your Microsoft account, make sure you've selected the correct
+        Microsoft account if you have multiple.
+        - Once the authorization is complete, you can download the Plain app for Teams (used in the next step).
+        - Optional: Authorize replies from your Teams profile to respond directly from Plain as yourself.
+    </Step>
+
+    <Step title="Add the Plain app to Teams">
+        <Warning>
+            The method of installation here will depend on how Teams is set up for your organisation. If you are having trouble
+            with setting this up, please get in touch so we can help!
```
Comment (jesushernandez) [inline microsoft-teams/installation.mdx:45]: Can we also use the `mailto` link here?

File: microsoft-teams/installation.mdx [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+        #### Option 1 - Install via Teams App
+        - Navigate to the Apps tab
+        - Click Manage your apps
+        - Click Upload an app
+        - As a Teams admin, you should be able to 'Upload an app to your organization's app catalogue' and upload the .zip file you downloaded in the previous step
+
+        #### Option 2 - Install via the [Teams Admin Dashboard](https://admin.teams.microsoft.com/dashboard)
+        - In the left sidebar, navigate to Teams apps -> Manage Apps
+        - Under Actions, click on 'Upload new app'
+        - Upload the .zip file you downloaded in the previous step
```
Comment (jesushernandez) [inline microsoft-teams/installation.mdx:57]: Some consistency in the section titles would be good. Eg, quoting them like you do with `'Upload new app'`. Either quote or emphasis (bold) would work. The latter makes it easier to read/follow IMO

File: microsoft-teams/installation.mdx [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+    </Step>
+
+</Steps>
+
+## Using the app
```
Comment (jesushernandez) [inline microsoft-teams/installation.mdx:62]: As a first time reader, it's not clear what app we're referring to: Plain, the MS Teams Plain App or MS Teams App?

**Pull request merged**